src/api/index.tsx

```diff
@@ -15,6 +17,8 @@ import {
   GRAPH_API_URL_DEVELOP_POLYGON,
   GRAPH_API_URL_DEVELOP_XDAI,
   GRAPH_API_URL_PRODUCTION_AVALANCHE,
+  GRAPH_API_URL_PRODUCTION_BASE_MAINNET,
+  GRAPH_API_URL_PRODUCTION_BASE_TESTNET,
```

Comment:
Do we need testnets on prod?

Reply:
In this case both are the same value but the distinction is provided for those who want to use one url while deving and another for builds